src/applications/mhv-medical-records/util/constants.js

```diff
@@ -131,7 +131,7 @@ export const vitalTypeDisplayNames = {
   BLOOD_PRESSURE: 'Blood pressure',
   PULSE: 'Heart rate',
   RESPIRATION: 'Breathing rate',
-  PULSE_OXIMETRY: 'Blood oxygen level (pulse oximetry)',
+  PULSE_OXIMETRY: 'Blood oxygen level',
```

Comment:
The ticket says to only remove "(pulse oximetry)" from the link, but to leave it in the `<h2>`. They both use the `displayName` constant so you'll have to special case it.

Reply:
Fixed this issue by parsing the link when string is received.